.gitattributes

```diff
@@ -5,3 +5,4 @@ java-checks/src/main/java/org/sonar/java/checks/helpers/ExpressionEvaluator.java
 java-checks-test-sources/src/main/java/checks/CounterModeIVShouldNotBeReusedCheck.java eol=lf
 java-checks/src/main/java/org/sonar/java/checks/helpers/HardcodedStringExpressionChecker.java eol=lf
 its/ruling/src/test/resources/autoscan/autoscan-diff-by-rules.json eol=lf
+java-checks-test-sources/src/main/files/non-compiling/checks/unused/UnusedPrivateMethod.java eol=lf
```

Comment:
Really not sure about this... Did you configure your git correctly to handle cr-lf appropriately?

https://github.com/SonarSource/sonar-developer-toolset#git